backend/models/office_hours/oh_event.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import BaseModel
+from datetime import datetime, date
+from oh_type import OfficeHoursType
+
+__authors__ = ["Sadie Amato"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class OfficeHoursEvent(BaseModel):
+    """
+    Pydantic model to represent an `OfficeHoursEvent` that has not been created yet.
+
+    This model is based on the `OfficeHoursEventEntity` model, which defines the shape
+    of the `OfficeHoursEvent` database in the PostgreSQL database.
+    """
+    id: int | None = None
+    office_hours_section_id: int
+    location_id: int
```

Comment:
Nit: I might be wrong here so feel free to ignore lol, but it might be good to rename this to `room_id`, since it is a foreign key to the `room` table! That way, later on, it is easier to understand which data / table this ID is referring to

Reply:
Updated, thank you!